src/niggli.c

```diff
@@ -125,8 +125,10 @@ int niggli_reduce_periodic(double *lattice_, const double eps_,
                            const int aperiodic_axis) {
     int i, j, succeeded;
     NiggliParams *p;
-    int (*steps[8])(NiggliParams * p) = {step1, step2, step3, step4,
-                                         step5, step6, step7, step8};
+    // clang-format off
```

Comment:
Lets avoid using `clang-format off` as much as possible. This one seems to be ab explicit pointer rule, and it should be adjustable. On my clion setup, I have it export all the rules, but I have different preferences (e.g. I prefer pointer symbol on the type, not on the value) so I can't share/generate mine.